automation/terraform/buildkite/buildkite-ci/us-east4.tf

```diff
@@ -68,7 +68,7 @@ locals {
         limits = {
           cpu    = "15.5"
           memory = "16G"
-          ephemeral-storage = "100Gi"
+          ephemeral-storage = "500Gi"
```

Comment:
Do we still need this bump with the build steps separation?

Reply:
No, I think the problem is not diskspace limit. I would undo this commit.